### PR TITLE
Added onPointerUp() and fixed an emit issue

### PR DESCRIPTION
**Pull request by shaneYeti**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # AUX Changelog
 
+## V0.9.28
+
+### Date: TBD
+
+### Changes:
+
+-   Improvements
+    -   Added the `onPointerUp()` action tag to fire on butotn release.
```
Comment (KallynGowdy) [inline CHANGELOG.md]: FYI button is typo'd here

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+-   Bug Fixes
+    -   Resolved issue where creating a new tag on one bot, deselecting all bots and sttempting to add that same tag to a different bot resulted in a warning.
```
Comment (KallynGowdy) [inline CHANGELOG.md]: also here with "attempting"

File: CHANGELOG.md (hunk 1 above)
Comment (shaneYeti) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (shaneYeti) [reply CHANGELOG.md]: Fixed

**Pull request merged**